### PR TITLE
Pull in slack_usergroup_channels.channels from both channels and groups

### LINKED ISSUE #44: Terraform plan cruft for slack_usergroup_channels resource
Slack's concepts of public and private channels are not unified on their backend. Though they are unified under the [conversations](https://api.slack.com/methods/conversations.info/test) API endpoints, they are "internally" represented as either "channels" (public channels) or "groups" (private channels); this is indicated by the `is_channel` and `is_group` booleans on the conversations model.

This distinction surfaces in the [user groups](https://api.slack.com/methods/usergroups.list/test) API endpoints in the `prefs.channels` and `prefs.groups` properties. This provider only examines the `prefs.channels` property when reading the usergroup, thus resulting in Terraform plan cruft because the resource thinks the private channels (i.e. "groups") are missing from the list. I haven't extensively tested this, but I believe the creation of the "usergroup_channel" relationship is working fine, regardless of this channel/group distinction.

### PR DESCRIPTION
**Pull request by rlmartin**
Resolves #44

### CODE REVIEW

**Comment by rlmartin** [comment]
I built the provider and tested locally. It produced a clean plan as I expected (previously, unchanged resources were being incorrectly identified requiring updates).

File: slack/resource_usergroup_channels.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func resourceSlackUserGroupChannels() *schema.Resource {
 				Required: true,
 			},
 			"channels": {
-				Type: schema.TypeList,
+				Type: schema.TypeSet,
```
Comment (jmatsu) [inline slack/resource_usergroup_channels.go:31]: Make sense.

File: slack/resource_usergroup_channels.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func resourceSlackUserGroupChannels() *schema.Resource {
 
 func configureSlackUserGroupChannels(d *schema.ResourceData, userGroup slack.UserGroup) {
 	d.SetId(userGroup.ID)
-	_ = d.Set("channels", userGroup.Prefs.Channels)
+	_ = d.Set("channels", append(userGroup.Prefs.Channels, userGroup.Prefs.Groups...))
```
Comment (jmatsu) [inline slack/resource_usergroup_channels.go:43]: I've confirmed https://api.slack.com/methods/usergroups.create and https://api.slack.com/methods/usergroups.update take just `channels` and they aren't distinguished.

**Review by jmatsu — APPROVED** [review]
Great. Thank you for your contribution.

**Comment by rlmartin** [comment]
Thanks! Will a new version get automatically released? Or will you do it manually at some point?

**Comment by rlmartin** [comment]
@jmatsu do you mind creating a release for this? Thanks!

**Pull request merged**